### PR TITLE
Simplify data model

### LINKED ISSUE #20: Change the serialization format
Flatbuffers is clearly overkill for the needs and quite painful to use.

We should should consider a better suited format

### PR DESCRIPTION
**Pull request by Tristramg**
Closes #20 #29

This pull request brings two improvements. Simplify the model to make it easier to explain and adopt and make it less error prone.

We might loose hypothetical use cases, but if they arise, that would mean that the rest is stable enough

### CODE REVIEW

**Review by louisgreiner — APPROVED** [review]
It was the right decision to simply indeed. Nice commit splitting.

File: schema/lrs.fbs [hunk 1]
```diff
@@ -99,72 +69,43 @@ table Anchor {
...
-    used_on:[TraversalRef];
+    traversal_index:uint32;
+    /// An LRM can apply to multiple traversal.
+    /// For instance, a LRM can be the central line of a highway.
+    /// And that LRM is the reference for the two other traversals corresponding to each direction.
+    used_on:[uint32];
     anchor_indices:[uint64] (required);
     distances:[double] (required);
-    /// The unit used to measure the distance between anchors
+    /// The unit used to measure the distance between anchorss
```
Comment (JulesPeignier) [inline schema/lrs.fbs:100]: Shouldn't it be just one s? 'anchors.' ?

File: src/lrs_generated.rs [hunk 2]
```diff
@@ -1998,15 +1538,15 @@ impl<'a> LinearReferencingMethod<'a> {
     // which contains a valid value in this slot
     unsafe { self._tab.get::<flatbuffers::ForwardsUOffset<flatbuffers::Vector<'a, f64>>>(LinearReferencingMethod::VT_DISTANCES, None).unwrap()}
   }
-  /// The unit used to measure the distance between anchors
+  /// The unit used to measure the distance between anchorss
```
Comment (JulesPeignier) [inline src/lrs_generated.rs:1541]: same as first comment

**Review by JulesPeignier — APPROVED** [review]

File: schema/lrs.fbs (hunk 1 above)
Comment (Tristramg) [reply schema/lrs.fbs]: good catch, it’s fixed. Thank you

**Pull request merged**